frontend/src/components/gridTemplate.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { PureComponent } from 'react'
+import Tiles from './ProjectTile'
+
+export default class gridTemplate extends PureComponent {
+    render() {
+        return (
+            <div className="grid grid-cols-6 pt-32 col-gap-2 row-gap-12">
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+                <Tiles/>
+            </div>
+        )
+    }
+}
```

Comment:
We can talk about this later, but a component normally is a single item aka one tile. The props that you'd pass to the tile would be the info relating to the item such as the title, number of songs and sets etc. Every component should be singularized aka Tile instead of Tiles. This whole gridTemplate file doesn't really serve a purpose

Reply:
For Visual testing purposes i just copied and paste the tiles, I'm hoping to do a loop when we get the database set up that just generates a tile for each Project.

Reply:
Yeah, my comment was regarding the gridTemplate component - it's not doing much, and when it comes to props it adds another layer. Say you wanted to do a loop to control the data that goes into the tiles, then because you've got Home.tsx which uses gridTemplate, which then uses Tiles, you'd have to pass as props twice the data. I would just loop the tiles within Home.tsx and remove this file

Reply:
okay, that makes sense. so i can add the div that's in the gridTemplate.tsx to the Home.tsx?

Reply:
Yeah that's totally fine

Reply:
nice one, ill update that now